### ISSUE TITLE
UNLIMITED PAUSE

### ISSUE THREAD

**Issue by jbartolome04** [ValveSoftware/Dota2-Gameplay#7036]
### Description

annoying UNLIMITED pause of either side even one player already quits. we are currently waiting about an hour. We are winning so the enemy team is annoying us out to quit so they can win the game. They are willing to pause it all day. battle of patience and time. F*ckn annoying dota.

### Screenshots

![image](https://user-images.githubusercontent.com/126067969/220590870-b0c3cfb1-a111-4e17-b7f7-1d7ce9139c43.png)

**Comment by jbartolome04** [comment]
![image](https://user-images.githubusercontent.com/126067969/220594419-4333829c-1149-46ce-88c0-f1bf15d73817.png)

**Comment by jbartolome04** [comment]
BAN THOSE SHITS

**Comment by shaileshgarg** [comment]
Yeah, it is annoying when the enemy can just resume/pause the game for an unlimited number of times.

Like in Pro games there is a fixed pause duration per team per game after which admin would forcefully start the game, something similar should be there for normal pub games. Like each team has a total of x mins of pause time, and beyond that they cannot pause and the game forcefully resumes.

**Comment by JAllied** [comment]
After that match. Was the match recorded or not? I experience it now it is so annoying. hopefully not recorded
